conf/grub.cfg

```diff
@@ -3,3 +3,4 @@
 #   set_global eve_flavor kvm
 # to force booting in Xen mode, uncomment:
 #   set_global eve_flavor xen
+set_getty
```

Comment:
Added by mistake. And yes, we need to change how we prepare the debug grub config. Otherwise it is really annoying.

Reply:
removed it
(I let this comment open in case I accidentally re-add it ...)